src/Library/demos/Toolbar View/main.js

```diff
@@ -0,0 +1,79 @@
+import Gio from "gi://Gio";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const topbar_select = workbench.builder.get_object("topbar_select");
+const bottombar_select = workbench.builder.get_object("bottombar_select");
+const toolbar_view = workbench.builder.get_object("toolbar_view");
+const reveal_topbar = workbench.builder.get_object("reveal_topbar");
+const reveal_bottombar = workbench.builder.get_object("reveal_bottombar");
+const header_top = workbench.builder.get_object("header_top");
+const header_bottom = workbench.builder.get_object("header_bottom");
+const action_bar = workbench.builder.get_object("action_bar");
+const switcher_bar = workbench.builder.get_object("switcher_bar");
+const popover = workbench.builder.get_object("popover");
+const search_bar = workbench.builder.get_object("search_bar");
+const gtk_box = workbench.builder.get_object("gtk_box");
+const tab_bar = workbench.builder.get_object("tab_bar");
+
+reveal_topbar.connect("notify::active", () => {
+  toolbar_view.reveal_top_bars = reveal_topbar.active;
+});
+
+reveal_bottombar.connect("notify::active", () => {
+  toolbar_view.reveal_bottom_bars = reveal_bottombar.active;
+});
+
+topbar_select.connect("notify::selected-item", () => {
+  if (topbar_select.get_selected() === 3) {
+    changeHeaderBar(workbench.builder.get_object("action_bar"));
+  } else if (topbar_select.get_selected() === 2) {
+    changeHeaderBar(workbench.builder.get_object("switcher_bar"));
+  } else if (topbar_select.get_selected() === 4) {
+    changeHeaderBar(workbench.builder.get_object("popover"));
+  } else if (topbar_select.get_selected() === 5) {
+    changeHeaderBar(workbench.builder.get_object("search_bar"));
+  } else if (topbar_select.get_selected() === 6) {
+    changeHeaderBar(workbench.builder.get_object("gtk_box"));
+  } else if (topbar_select.get_selected() === 1) {
+    changeHeaderBar(workbench.builder.get_object("tab_bar"));
+  }
```

Comment:
Not sure if we'll keep this, but anything like this should use a `switch` statement.

Reply:
Changed to switch statement